### PR TITLE
moving analytics escape hatch to its own section

### PR DESCRIPTION
**Pull request by kneekey23**
*Description of changes:* Making analytics escape hatch easy to find by moving to its own tab under the category.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/lib/analytics/escapehatch.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+title: Escape hatch
+description: For advanced use cases where Amplify does not provide the functionality, you can retrieve the escape hatch to access the AWSS3 instance.
```
Comment (lawmicha) [inline docs/lib/analytics/escapehatch.md:3]:
```suggestion
description: For advanced use cases where Amplify does not provide the functionality, you can retrieve the escape hatch to access the AWSPinpoint instance.
```

File: docs/lib/analytics/fragments/android/escapehatch.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+For advanced use cases where Amplify does not provide the functionality, you can retrieve the escape hatch to access AWSPinpoint instance.
+
+<amplify-block-switcher>
+<amplify-block name="Java">
+
+```java
+AWSPinpointAnalyticsPlugin plugin = (AWSPinpointAnalyticsPlugin) Amplify
+        .Analytics
+        .getPlugin("amazonPinpointAnalyticsPlugin");
```
Comment (lawmicha) [inline docs/lib/analytics/fragments/android/escapehatch.md:9]: is this correct on android? i think it should be `awsPinpointAnalyticsPlugin`

File: docs/lib/analytics/menu.json [hunk 3]
```diff
@@ -7,6 +7,7 @@
     "identifyuser",
     "streaming",
     "storing",
-    "personalize"
+    "personalize",
+    "escapehatch"
```
Comment (lawmicha) [inline docs/lib/analytics/menu.json:11]: we duplicate the list of sections from the menu in the getting-started page, so we should add the escapehatch page link at the bottom of the getting started guide as well

File: docs/lib/analytics/fragments/android/escapehatch.md (hunk 2 above)
Comment (kneekey23) [reply docs/lib/analytics/fragments/android/escapehatch.md:9]: I just moved this I didnt change it?

File: docs/lib/analytics/menu.json (hunk 3 above)
Comment (kneekey23) [reply docs/lib/analytics/menu.json:11]: duplicated

File: docs/lib/analytics/fragments/android/escapehatch.md (hunk 2 above)
Comment (kneekey23) [reply docs/lib/analytics/fragments/android/escapehatch.md]: fixed this

File: docs/lib/analytics/fragments/android/escapehatch.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+For advanced use cases where Amplify does not provide the functionality, you can retrieve the escape hatch to access AWSPinpoint instance.
```
Comment (jpignata) [inline docs/lib/analytics/fragments/android/escapehatch.md:1]:
```suggestion
For advanced use cases where Amplify does not provide the functionality, you can retrieve the escape hatch to access the underlying Amazon Pinpoint client.
```

File: docs/lib/analytics/fragments/ios/escapehatch.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
+For advanced use cases where Amplify does not provide the functionality, you can retrieve the escape hatch to access AWSPinpoint instance.
```
Comment (jpignata) [inline docs/lib/analytics/fragments/ios/escapehatch.md:1]:
```suggestion
For advanced use cases where Amplify does not provide the functionality, you can retrieve the escape hatch to access the underlying Amazon Pinpoint client.
```

**Pull request merged**